<launch>

  <arg name="model" default="burger"/>
  <node name="rviz1" pkg="rviz" type="rviz" args="-d $(find ros_topoexplore)/rviz/topoexplore.rviz" />
  <node name="rviz2" pkg="rviz" type="rviz" args="-d $(find ros_topoexplore)/rviz/topoexplore_two.rviz" />


  <node name="map_building1" pkg="ros_topoexplore" type="map.py" output="screen">
    <param name="network" value="gl18-tl-resnet101-gem-w" />
    <param name="platform" value="cuda" />
    <param name="robot_name" value="robot1"/>
    <param name="robot_num" value="2"/>
  </node>

  <node name="map_building2" pkg="ros_topoexplore" type="map.py" output="screen">
    <param name="network" value="gl18-tl-resnet101-gem-w" />
    <param name="platform" value="cuda" />
    <param name="robot_name" value="robot2"/>
    <param name="robot_num" value="2"/>
  </node>

  <node name="experiment" pkg="ros_topoexplore" type="experiment.py" output="screen">
    <param name="robot_num" value="2"/>
  </node>

</launch>